### ISSUE TITLE
Playback control not supported

### ISSUE THREAD

**Issue by D3rpi** [alexa/alexa-avs-sample-app#976]
I updated my the code to the newest version this weekend and also enabled the reminders through my developer account but now I get this error in the web app.
 I can no longer view my tunein stations, my alarms nor the now playing through the web app.

> Playback control not supported
> To control playback, speak to *****'s Amazon Apps and tell Alexa what you'd like to do.
>

**Comment by BennyAvramson** [comment]
Hi @D3rpi,

If I understood you correctly, the errors you report are on the Alexa website. 
It doesn't look like it has something to do with the sample app. 
I suggest you to ask the question on [Amazon Developer Alexa forum](https://forums.developer.amazon.com/spaces/165/alexa.html)

Thanks,
Benny

**Comment by D3rpi** [comment]
@BennyAvramson Yes, I believe it's a website issue. I did post there too, but haven't heard anything yet.
